region/norfair/east/Bubble Mountain.json

```diff
@@ -1235,6 +1547,45 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 7],
+      "name": "Come in Getting Blue Speed, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 0,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$2.6"
```

Comment:
This gets really hard to save if your run speed if not even that much higher.
With a little more speed than a 3 tap, you don't get the height. And a little more than that reaches the wall before being high enough to clear it.  And a pause remorph is very difficult to do there.

Reply:
Yeah this needed to have a `maxExtraRunSpeed`. Added that now. With a range of $2.6 to $2.F, it's a 10-frame window in terms of how many frames to hold dash. Above $2.F there's a dropoff in jump speed which would make it not work. We could add a `canTrickyDashJump` if you think that would help? This one is not as precise as the 2-frame window which that tech can imply in the worst case. I have been thinking it would be good to split off an easier version of that tech, to go into Very Hard, for jumps that are tricky but more lenient than 2-frame precise. In any case, this strat has `canLongChainTemporaryBlue` which is a high-difficulty tech which we could maybe assume to include a fairly high level of movement in general, even more so than `canChainTemporaryBlue`?